subjects/shoppingsummarycounter/README.md

```diff
@@ -0,0 +1,46 @@
+## shoppingsummarycounter
+
+### Instructions
+
+You have a receipt from the grocery store and you want to know how many of each item you bought. Write a function that returns this summary.
+
+Given a `string` count the total amount of times each item appears in it and return a summary including the item and its number of appearances as an `int`.
+
+### Expected function
+
+```go
+func ShoppingSummaryCounter(str string) map[string]int {
+
+}
+```
+
+### Usage
+
+Here is a possible program to test your function:
+
+```go
+package main
+
+import (
+  "fmt"
+  "piscine"
+)
+
+func main() {
+  summary := "Burger Water Carrot Coffee Water Water Chips Carrot Carrot Burger Carrot Water"
+    for index,element := range piscine.ShoppingSummaryCounter(summary){
```

Comment:
After a `,` there should be a space. Try putting this into a go  file and run gofmt. There should be a space before the opening curly brace as well.

Reply:
fixed